### ISSUE TITLE
move aria from api to feature

### ISSUE THREAD

**Issue by mattgarrish** [w3c/a11y-discov-vocab#4]
I'm sure we've discussed this before, but ARIA isn't an accessibility API but more of a bridging technology between semantic-less HTML and the APIs.

The accessibilityAPI property is more for describing whether reading systems behave with the platform APIs, so isn't something we should be setting for publications. If we move the ARIA value to the features list, we could omit any need for this property (for publications, not suggesting we drop it).

That said, we might want to replace it with something more generic like `accessibleScripting` so that we can identify all dynamic content that is accessible, not just content that has to use ARIA.
